testutils/infra/device.py

```diff
@@ -110,7 +110,20 @@ def ssh_is_opened(self, wait=60 * 60):
         Keyword arguments:
         wait - Timeout (in seconds)
         """
-        self.run("true", hide=True, wait=wait)
+        waited = -1
```

Comment:
How about we use the `redo` package here as well?

https://github.com/mendersoftware/integration/blob/master/testutils/infra/container_manager/docker_compose_base_manager.py#L21

Reply:
done.